AppInspector/Commands/VerifyRulesCommand.cs

```diff
@@ -102,13 +103,14 @@ public VerifyRulesResult GetResult()
                 RequireMustMatch = _options.RequireMustMatch,
                 RequireMustNotMatch = _options.RequireMustNotMatch,
                 LoggerFactory = _loggerFactory,
-                LanguageSpecs = Languages.FromConfigurationFiles(_loggerFactory, _options.CustomCommentsPath,
-                    _options.CustomLanguagesPath)
+                LanguageSpecs = Languages.FromConfigurationFiles(_loggerFactory, _options.CustomCommentsPath, _options.CustomLanguagesPath),
+                EnableNonBacktrackingRegex = _options.EnableNonBacktrackingRegex,
             };
             RulesVerifier verifier = new(options);
             verifyRulesResult.ResultCode = VerifyRulesResult.ExitCode.Verified;
 
-            RuleSet? ruleSet = new(_loggerFactory);
+            RuleSet? ruleSet = new(_loggerFactory) { EnableNonBacktrackingRegex = options.EnableNonBacktrackingRegex };
+
             if (_options.VerifyDefaultRules)
```

Comment:
Should add a specific check in here for incompatible `nb` and `b` both specified - and cases with Regex with `nb` explicitly set that cannot be constructed as `nb` (but not those which cannot be constructed with `nb` but only because the `EnableNonBacktrackingRegex` option was set.